ml-proto/host/import/spectest.ml

```diff
@@ -0,0 +1,12 @@
+open Types
+
+
```

Comment:
A top-level comment would be useful here as well. Something like "The spectest module provides functions that support the specialized testsuite execution environment."